.github/workflows/docker.yml

```diff
@@ -23,11 +33,18 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+        with:
+          # Check out pull request's HEAD commit instead of the merge commit.
+          ref: ${{ github.event.pull_request.head.sha }}
       - name: Determine tag name
         id: determine_tag
         shell: bash
         run: |
-          branch=${GITHUB_REF#refs/heads/}
+          if [[ -z $GITHUB_BASE_REF ]]; then
```

Comment:
Please add a comment explaining when `if` and `else` are supposed to be triggered